### DISCUSSION TITLE
Current data lake (influxdb) capacity usage

### DISCUSSION THREAD

**Discussion by yanfeng1992** [apache/streampipes#2614] [Ideas]
Currently, streampipe does not provide the usage of data lake (influxdb) data storage capacity. Can a function be added to display the current data storage usage capacity and remaining data capacity of the data lake (influxdb)?

for example

Total capacity: 100G
Currently used capacity: 80G
Remaining capacity: 20G

@dominikriemer 

![image](https://github.com/apache/streampipes/assets/89568107/f4a271ed-f031-401c-948e-d5e6b2e663b7)

**Comment by tenthe** [comment]
Hi @yanfeng1992,

I really like this suggestion to give more feedback to the user.
We already have some information about the host machine under `User Settings` -> `Info` -> `System Info`, but I think this is not enough and would be great to get direct information about the disk usage of the time series database.

The question is how we can achieve this in a generic way, since the system can run in either Docker or Kubernetes. There are also ongoing efforts to replace the InfluxDB with an IoTDB. This means that the approach should work for both technologies.

Do you or anyone else in the community already have an idea for a suitable approach?

Cheers,
Philipp
